### ISSUE TITLE
T2.2 - Development User Profile APIs to be consumed by FE (create endpoint)

### ISSUE THREAD

**Issue by riccardo-leoni-dw** [Gravitate-Health/MVP2#11]
Form mapped to the BE APIs to create the user profile

**Comment by amedranogil** [comment]
@abelmarm please provide plan.
I think we can start with simple Loopback CRUD using the current profile as model.

**Comment by Paride07** [comment]
@paolopierantozzi please check this task

**Comment by Paride07** [comment]
@paolopierantozzi reported after an internal call with UPM that the patient profile is not linked to the patient account and that UPM will release CRUD associated with the patient account at end of February alongside the authentication WRAPPER.
@amedranogil and @abelmarm could you confirm this?

**Comment by abelmarm** [comment]
That is right, the link will be via a unique identifier, but will not be the same object.

**Comment by Paride07** [comment]
Update 20.02.23
To be postpone to Sprint 3

**Comment by Paride07** [comment]
Update 27.02.23
status: ready to be reviewed @paolopierantozzi provide feedback

**Comment by paolopierantozzi** [comment]
Initial review update 27.03.23:
The openapi interface seems good, but there is the need for the postman collection and the authentication layer in order to provide a complete feedback on the effective working status of the APIs.

Also, for the moment alergies, diagnoses, intolerances and medicines are treated as strings, while this could be a viable solution for testing, in a real deployment environment it's mandatory to treat them as complete FHIR resources/ json
